viewer/src/components/PlayerCircle.vue

```diff
@@ -12,6 +12,7 @@ import { Component, Prop, Vue } from "vue-property-decorator";
 import Engine, { factions, Phase, Planet, Player, PlayerEnum } from "@gaia-project/engine";
 import { phaseBeforeSetupBuilding } from "../logic/utils";
 import { factionPlanet } from "@gaia-project/engine/src/factions";
+import { AuctionVariant } from "@gaia-project/engine/src/engine";
```

Comment:
This should be directly exported/imported from `"@gaia-project/engine"` (and I'm noticing `factionPlanet` should as well 😬 )